### ISSUE TITLE
[WR] Function names evaluated as types

### ISSUE THREAD

**Issue by dantleech** [phpactor/phpactor#232]
```
        $templateBody = file_get_contents($this->template);
```
```
{
    "symbol": "replaceTokens",
    "symbol_type": "method",
    "start": 2286,
    "end": 2299,
    "type": "string",
    "container_type": "PhpBench\\Benchmark\\Remote\\Payload",
    "value": "NULL",
    "offset": 2294,
    "type_path": null,
    "frame": {
        "locals": {
            "1971": "$this = (PhpBench\\Benchmark\\Remote\\Payload) NULL",
            "2218": "$templateBody = (PhpBench\\Benchmark\\Remote\\file_get_contents) NULL",
            "2325": "$scriptPath = (PhpBench\\Benchmark\\Remote\\tempnam) NULL",
            "2438": "$wrapper = (string) ''",
            "2495": "$wrapper = (<unknown>) NULL",
            "2990": "$output = (<unknown>) NULL",
            "3037": "$result = (PhpBench\\Benchmark\\Remote\\json_decode) NULL"
        }
    }
}
```

**Comment by dantleech** [comment]
The function is a fully qualified name, and we always assume that is a type.

I guess we could identify that it is a function if:

- The parent node is not a class creation expression.
- The parent node is not a scope property access (static call).
- .... ?

**Comment by dantleech** [comment]
Fixed in e8eb8c8
